src/components/base/outline-tooltip/outline-tooltip.css

```diff
@@ -0,0 +1,41 @@
+:host {
+  @apply inline-flex flex-col;
+}
+
+.tooltip {
+  @apply flex relative justify-center;
+}
+
+.tooltip-tip {
+  @apply absolute inline-flex p-2 opacity-0 text-center;
+  visibility: hidden;
+  pointer-events: visible;
+  border: black solid 1px;
+  background-color: whitesmoke;
+  transition: 300ms;
+  transition-delay: 800ms;
+}
+
+.position-top {
+  @apply bottom-full mb-2;
+}
+
+.position-bottom {
+  @apply top-full mt-2;
+}
+
+.position-right {
+  @apply left-full ml-2;
+}
+
+.position-left {
+  @apply right-full mr-2;
+}
+
+.tooltip:hover > .tooltip-tip,
+.tooltip:focus > .tooltip-tip {
+  @apply opacity-100;
+  visibility: visible;
```

Comment:
As above we can use `visible` in the `@apply` directive to handle the `visibility` property here

Reply:
^^

Reply:
noted, thanks for the clarification